### PR TITLE
Add more italian data

### PR DESCRIPTION
**Pull request by miscelatore**
Hi,

I submit my changes concerning the support for the Italian language of some modules.

Regards.
--
Stefano

### CODE REVIEW

File: lib/ffaker/gender_pl.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ module GenderPL
     extend ModuleUtils
     extend self
 
-    GENDERS = %w[kobieta mężczyzna].freeze
+    GENDERS = %w[donna uomo].freeze
```
Comment (Volosh1n) [inline lib/ffaker/gender_pl.rb:10]: That's was Polish

Comment (miscelatore) [reply lib/ffaker/gender_pl.rb]: Sorry I fix this.

**Comment by miscelatore** [comment]
Sorry I fix this.

**Comment by Volosh1n** [comment]
But why you closed it?

**Comment by miscelatore** [comment]
Hi re-submit a new pull-request after fix the file gender_pl.rb

Il giorno sab 7 ago 2021 alle ore 14:47 Vsevolod Voloshyn <
***@***.***> ha scritto:

**Comment by miscelatore** [comment]
I also added tests of: address_it, gender_it, identification_it,
job_it, lorem_it
and fix test_units.rb

Il giorno sab 7 ago 2021 alle ore 14:47 Vsevolod Voloshyn
***@***.***> ha scritto:

**Comment by Volosh1n** [comment]
New PR: #501

**Pull request closed without merging**